### DISCUSSION TITLE
unable to import DOMHelper from glsp/client

### DISCUSSION THREAD

**Discussion by rileydanejohnston** [eclipse-glsp/glsp#843] [General] [answered]
Hello, 
I’m trying to create a UI extension similar to editLabelUI, which uses a class called DOMHelper that I am unable to import from @eclipse-glsp/client. I wanted to see if this class is intentionally not exported, or of there was a way around this. Thank you!

**Comment by tortmayr** [comment, marked as answer]
Hi,
the `@eclipse-glsp/client` only reexports the main index file of sprotty. This main index file does not necessarily export all symbols/classes/types. Apparently the `DOMHelper` class is not included. I don't know the reasoning behind this (could be intentionally or a mere oversight). As a workaround you can import the `DOMHelper` class using the complete path:
```ts
import { DOMHelper } from 'sprotty/lib/base/views/dom-helper';
```

**Reply by rileydanejohnston** [reply]
This is worked for me. Thank you @tortmayr!
